docs/MockFunctionAPI.md

```diff
@@ -526,13 +526,17 @@ test('calculate calls add', () => {
 The `jest.Mocked<Source>` utility type returns the `Source` type wrapped with type definitions of Jest mock function.
 
 ```ts
-import fetch from 'node-fetch';
 import {expect, jest, test} from '@jest/globals';
+import type {fetch} from './fetch';
```

Comment:
this looks wrong - a type import will fail the `typeof` below. But why not keep the `node-fetch` import?

Reply:
Hm.. Checked it. `type` works, TS does not complain. Similar to the above: https://github.com/facebook/jest/blob/2f4340cb1da435a9b147e238f7034f99653b8070/docs/MockFunctionAPI.md?plain=1#L500-L505

I was not happy that ESLint (`import/order`) is pushing to keep `'fetch'` import above `'@jest/globals'`. Probably some ignore comment could fix this.